src/component/Context/WishListContext/WishListContext.js

```diff
@@ -1,49 +1,88 @@
-import { useContext, useState, createContext } from "react";
+import { useContext, useState, createContext, useReducer } from "react";
```

Comment:
Suggestion : Code for reducer and api calls can be extracted out .

Reply:
Thanks for suggestion will do it in next PR